src/resolvers/ResolverBuilder.ts

```diff
@@ -100,7 +105,9 @@ export class ResolverBuilder {
       schemaDefinition: `${fieldName}(fields: QueryFilter): [${gqlType.name}]!`,
       action: ResolverType.FIND,
       resolverType: "Query",
-      implementation: `return ${this.knexContext}.select().from('${tableName}').where(${this.argumentContext}.field.name, '=', ${this.argumentContext}.field.value)`
+      implementation: `(root, args, context, info) => {
+        return ${this.knexContext}.select().from('${tableName}').where(${this.argumentContext}.field.name, '=', ${this.argumentContext}.field.value)
```

Comment:
This is not properly implemented. Let's create separate issues for pagination and filtering as they are simply not there.

Reply:
Yes, the way they are defined now, they are unable to run as some of the stuff are undefined.
 `db` for instance in the implementations.

Reply:
Yes.. That is why I recomend you to take look into showcase and remove implementation specific stuff from it. 
It has db mounted. Take look ionic-showcase/server has everything and we should probably push that first. If you need help on this I can also come over and do it

Reply:
Let's have quick chat about this.